apps/judicial-system/backend/src/app/modules/file/file.controller.ts

```diff
@@ -143,6 +144,7 @@ export class FileController {
     judgeRule,
     registrarRule,
     assistantRule,
+    defenderRule,
```

Comment:
Not sure if this is the best way to open verdict access to defenders as it gives defenders access to all case files. Lets discuss.